### PR TITLE
Lint CiviCRM Rules module

### PR DESCRIPTION
**Pull request by seamuslee001**

### CODE REVIEW

**Comment by seamuslee001** [comment]
Jenkins test this please

**Comment by seamuslee001** [comment]
Jenkins test this please

**Comment by seamuslee001** [comment]
Jenkins test this please

File: modules/civicrm_rules/civicrm_rules.module [hunk 1]
```diff
@@ -33,13 +33,6 @@
  *
  */
 
-/*
```
Comment (seamuslee001) [inline modules/civicrm_rules/civicrm_rules.module:36]: Movement of this is as per "The hook is called every time the module is enabled. It should be implemented in the module's .install file. The implementation can rely on the .module file being loaded."

From http://www.drupalcontrib.org/api/drupal/drupal%21modules%21system%21system.api.php/function/hook_enable/7

**Comment by eileenmcnaughton** [comment]
I eyeballed these & the only functional change is moving the enable function to the install file, which makes sense

**Pull request merged**